plugins/wasi_cli/base.h

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2024 Second State INC
+
+#pragma once
+
+#include "env.h"
+
+#include "common/errcode.h"
+#include "runtime/hostfunc.h"
+
+namespace WasmEdge {
+namespace Host {
+
+template <typename T>
+class WasiCli : public Runtime::Component::HostFunction<T> {
```

Comment:
Comment:
Maybe add a namespace `Preview2`.
Since there will be preview 3 in the future.

Reply:
Agree